src/com/palmergames/bukkit/towny/command/BaseCommand.java

```diff
@@ -360,4 +360,15 @@ public static void checkPermOrThrow(Permissible permissible, String node) throws
 	public static void checkPermOrThrowWithMessage(Permissible permissible, String node, Translatable errormsg) throws NoPermissionException {
 		TownyUniverse.getInstance().getPermissionSource().testPermissionOrThrow(permissible, node, errormsg);
 	}
+
+	public static List<String> getTrustedTownsFromResident(Player player){

```

Comment:
```suggestion
	public static List<String> getTrustedTownsFromResident(Entity player){

```
Type of parameter 'player' may be weakened to 'org.bukkit.entity.Entity'

Reply:
We wont ever be passing on an entity and we wouldn't want to.

Reply:
We just use the functionality of an Entity so it should be weekend referring to Java Conventions. A Player isn't necessary here. We can then check if the Entity can be casted to a Player to check If the giving entity is a player, but this is also kind of unnecessary.